TestAssemblies/SimpleTest/TestRecords.cs

```diff
@@ -12,7 +12,7 @@ public enum Method {
     }
 
     public static class TestRecords {
-        private static readonly IList<Tuple<int, object[]>> _records = new List<Tuple<int, object[]>>();
```

Comment:
I would like to use .net version preprocessor for this. Like in my comment for TestBase.